engine-tests/build.gradle

```diff
@@ -80,8 +80,40 @@ task copyResourcesToClasses(type:Copy) {
 }
 
 //TODO: Remove it  when gestalt will can to handle ProtectionDomain without classes (Resources)
-test.dependsOn copyResourcesToClasses
-test.dependsOn rootProject.extractNatives
+test {
+    dependsOn copyResourcesToClasses
+    dependsOn rootProject.extractNatives
```

Comment:
I could _not_ figure out what that comment was referring to last time I looked at this PR, but I get it now. It's specifically that `dependsOn(copyResourcesToClasses)`.

Suggest moving the comment inside the braces to make that clearer.